tests/test-war-integration/src/main/resources/logging.properties

```diff
@@ -1,5 +1,6 @@
 com.google.cloud.runtimes.level = FINE
-com.google.cloud.runtimes.handlers = com.google.cloud.logging.LoggingHandler
+# Ideally we would prefer to set the handler for a specific class however this is not functional yet.
```

Comment:
Can you put in a link to the issue filed against the client library so that we know when we can change it back? Also, leave `#com.google.cloud.runtimes.handlers = com.google.cloud.logging.LoggingHandler` in -- later to be switched back.